src/main/kotlin/com/hedvig/rapio/qa/QaMemberServiceClient.kt

```diff
@@ -0,0 +1,20 @@
+package com.hedvig.rapio.qa
+
+import com.hedvig.rapio.externalservices.memberService.dto.IsMemberRequest
+import com.hedvig.rapio.qa.dto.MemberServiceUnsignMemberRequest
+import com.hedvig.rapio.qa.dto.UnsignMemberRequest
+import org.springframework.cloud.openfeign.FeignClient
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+
+@FeignClient(
+    name = "member-service",
```

Comment:
Unsure how FeignClient works, but can we have two with the same name, and the same URL?
```suggestion
    name = "member-service",
```

Reply:
Seems like it, I did not run into any problems when running locally